src/ccache/args.cpp

```diff
@@ -68,6 +72,8 @@ Args::from_atfile(const std::string& filename, AtFileFormat format)
     switch (*pos) {
     case '\\':
       switch (format) {
+      default:
+        break;
```

Comment:
Suggestion: list all enum values instead of defaulting, e.g.:

```suggestion
      case guess_from_compiler:
        ASSERT(false); // Can't happen
        break;
```